clang-tools-extra/clang-doc/BitcodeReader.cpp

```diff
@@ -681,6 +683,7 @@ llvm::Error ClangDocBitcodeReader::readRecord(unsigned ID, Reference *I) {
 // Read a block of records into a single info.
 template <typename T>
 llvm::Error ClangDocBitcodeReader::readBlock(unsigned ID, T I) {
+  llvm::TimeTraceScope("readBlock", "ClangDocBitcodeReader");
```

Comment:
there's really no logic I've updated the pr, so that it follow the convention of naming it after the scope or method the timescope is in

Reply:
Have you looked at the profile with the new names? Is it easy to follow? I’d expect it’s a bit harder now, since you’ve dropped some info. 

From what I can see, the typical usage of the name is something like “parse input files”. Mostly within our tools we seem to try to capture how long some task is taking, e.g. parsing files, constructing the ast, generating call graphs, writing out files, etc. 

I’d consider structuring these more around the tasks your tracking, and perhaps use the details to track more specific bits within a larger task or phase.

Reply:
I renamed most the labels to follow the convention of [task we are tracking] -> [details]